.circleci/config.yml

```diff
@@ -178,6 +197,8 @@ jobs:
     executor: test-executor
     steps:
       - checkout
+      - setup_remote_docker
```

Comment:
not needed for test executor as we don't build or push it:
https://circleci.com/docs/2.0/building-docker-images/

> once setup_remote_docker is called, a new remote environment is created, and your primary container is configured to use it. All docker-related commands are also executed in your primary container, but building/pushing images and running containers happens in the remote Docker Engine. (line 10)